### PR TITLE
✨ Add welcome-page

### PR DESCRIPTION
**Pull request by riittagerhards**
Add first page

image

### CODE REVIEW

File: src/app/App.tsx [hunk 1]
```diff
@@ -1,7 +1,45 @@
...
+          />
+          <input
+            className={styles.textinput}
+            type="text"
+            placeholder="last name"
+          />
+          <input
+            className={styles.submitbutton}
+            type="submit"
+            value="register"
```
Comment (AnkePfei2fer) [inline src/app/App.tsx]: this doesn't seem to work, try: ` value={"register"}`

File: src/app/App.module.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
+.container {
+  display: grid;
```
Comment (AnkePfei2fer) [inline src/app/App.module.css:2]: I would suggest adding a small gap here in order to show the lower border of your submit button correctly (I hope that works...)

**Review by AnkePfei2fer — COMMENTED** [review]
lgtm! Great work!

File: src/app/App.tsx (hunk 1 above)
Comment (riittagerhards) [reply src/app/App.tsx]: sorry, old picture in pull request. Code works :)

Comment (AnkePfei2fer) [reply src/app/App.tsx]: Sorry, I was wrong about that

**Pull request merged**